README.md

```diff
@@ -0,0 +1,198 @@
+  cd my-folder
+  https://github.com/Fernando426/Portfolio-mobile-website.git
+
+### Install
+
+*Install this project with:
+- Any code editor
+- Node.js
+- Git
+- GitHub
+
+### Usage
+
+To run the project, execute the following command:
+
+you just have to double click on index.html file
+
+### Run tests
+
+To run tests, run the following command:
+
+you can run and verify the correct functioning of the linters from the command console using:
+npx stylelint "**/*.{css,scss}"
+
+to run the project you just have to double click on the file index.html
+
+### Deployment
+
+You can deploy this project using:
+
+-Git
+-Github
+-The text editor of your choice
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+👤 **Ali Akbar Sazish**
+- GitHub: [@githubhandle](https://github.com/akbarsazish)
+- Twitter: [@twitterhandle](https://twitter.com/AliAkbarSazish1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ali-akbar-sazish/)
+
+👤 **Fernando Montes**
+
+- GitHub: [@Fernando426](https://github.com/Fernando426)
+- Twitter: [@Fernando_426_](https://twitter.com/Fernando_426_)
+- LinkedIn: [Fernando Montes](https://www.linkedin.com/in/fernando-jose-montes-linares-523822270/?originalSubdomain=ve)
+
+👤 **Jhonatan Caamal Erosa**
+
+- GitHub: [@JhonatanErosa](https://github.com/JhonatanErosa)
+- Twitter: [@erosa_c](https://twitter.com/erosa_c)
+- LinkedIn: [Jhonatan Ali Caamal Erosa](https://www.linkedin.com/in/jhonatan-ali-caamal-erosa-bb3829271/)
+
+👤 **José De La Rosa**
+
+- GitHub: [@jrraul4](https://github.com/jrraul4)
+- Twitter: [@josedelarosa_1](https://twitter.com/joserauldelaro1)
+- LinkedIn: [José De La Rosa](https://www.linkedin.com/in/jose-raul-de-la-rosa-98aa70265/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[new sections]**
+- [ ] **[interactive content]**
+- [ ] **[collaboration options]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+With your help we can keep moving forward, thank you!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, I would like to invite you to show your support for the growth and development of professional Web programming, any contribution, comment or suggestion will be very well received!
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to first thank the team and the platform as such at Microverse for making this fun, enjoyable and useful phase of a whole path of preparation towards world-class professionalism and training. also to the people, coding partners, guides and everyone who in one way or another has contributed to the advancement of the entire program. thank you so much!!!
```

Comment:
- [ ]  Don't forget to also give a shoutout in the acknowledgments section to the talented original designer [Cindy Shin](https://www.behance.net/adagio07?log_shim_removal=1) on Behance. Their amazing work deserves some recognition!